### PR TITLE
Generate changelog for 2.11.1

### PR DESCRIPTION
**Pull request by kyukhin**
Also, remove RC-related changelog and unreleased/ entries.

NO_DOC=no code changes
NO_TEST=no code changes
NO_CHANGELOG=no code changes

### CODE REVIEW

File: changelogs/2.11.1.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+#### Cfg
+
+* Implemented a way to set a table as box.cfg{} options value via
+  environment variables (gh-8051).
+
+### Test
+
+#### Fuzz
+
+* LuaJIT now can be fuzzed using grammar-based fuzzer (gh-4823).
```
Comment (andreyaksenov) [inline changelogs/2.11.1.md:66]:
```suggestion
* LuaJIT now can be fuzzed using a grammar-based fuzzer (gh-4823).
```

File: changelogs/2.11.1.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+  IPROTO key. The bug resulted in broken connectivity between Tarantool 2.10
+  and 2.11 (gh-8745).
+* Fixed the node writing an empty `00000000000000000000.xlog` file regardless of
+  the actual vclock when interrupted during the initial `box.cfg()` call
+  (gh-8704).
+* Fixed a bug causing the `ER_CURSOR_NO_TRANSACTION` failure for transactions
+  on synchronous spaces when the `on_commit/on_rollback` triggers are set
+  (gh-8505).
+* Fixed a bug causing the effective session and user are not propagated to
+  `box.on_commit` and `box.on_rollback` trigger callbacks when transaction
```
Comment (andreyaksenov) [inline changelogs/2.11.1.md:112]:
```suggestion
  `box.on_commit` and `box.on_rollback` trigger callbacks when the transaction
```

File: changelogs/2.11.1.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+  is synchronous (gh-8742).
+* Fixed a bug when MVCC sometimes lost gap record (gh-8326).
```
Comment (andreyaksenov) [inline changelogs/2.11.1.md:114]:
```suggestion
* Fixed a bug when MVCC sometimes lost a gap record (gh-8326).
```

File: changelogs/2.11.1.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+* Fixed a bug when MVCC rollback of prepared statement could break internal
```
Comment (andreyaksenov) [inline changelogs/2.11.1.md:115]:
```suggestion
* Fixed a bug when MVCC rollback of a prepared statement could break internal
```

File: changelogs/2.11.1.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+* Now the `ARRAY`, `MAP`, and `INTERVAL` values cannot be used as
+  arguments in the `ORDER BY` clause (gh-6668).
+* Fixed incorrect conversion of an integer greater than `INT64_MAX` or
+  less than `0` to a decimal number during SQL arithmetic operations (gh-8460).
+* Fixed an assertion when selecting tuples with incomplete internal
+  format (gh-8418).
+* Fixed integer overflow issues in built-in functions (ghs-119).
+* Fixed a possible assertion or segmentation fault when optimizing
+  `INSERT INTO ... SELECT FROM` (gh-8661).
+* Fixed an integer overflow issue and added check for the `printf()` failure due
```
Comment (andreyaksenov) [inline changelogs/2.11.1.md:242]:
```suggestion
* Fixed an integer overflow issue and added a check for the `printf()` failure due
```

File: changelogs/2.11.1.md (hunk 2 above)
Comment (kyukhin) [reply changelogs/2.11.1.md]: Fixed.

File: changelogs/2.11.1.md (hunk 3 above)
Comment (kyukhin) [reply changelogs/2.11.1.md]: Fixed.

File: changelogs/2.11.1.md (hunk 4 above)
Comment (kyukhin) [reply changelogs/2.11.1.md]: Fixed.

File: changelogs/2.11.1.md (hunk 5 above)
Comment (kyukhin) [reply changelogs/2.11.1.md]: Fixed.

**Pull request merged**